examples/complete/config/cert-manager.yaml

```diff
@@ -61,9 +61,3 @@ cainjector:
 
   podAnnotations:
     co.elastic.logs/enabled: "true"
-prometheus:
```

Comment:
Is there specific need to remove Prometheus monitoring?

Reply:
The Prometheus block is moved to the internal value.yaml file (modules/cert-manager/config/values.yaml) and removed from examples/complete/config/cert-manager.yaml